CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Fixes
 - Fix nesting issues with `Dialog` component inside `Popup` @layershifter ([#1706](https://github.com/stardust-ui/react/pull/1706))
+- Fix `Menu` and `MenuItem` styles to match [redlines] @bcalvery ([#1712](https://github.com/stardust-ui/react/pull/1712))
```

Comment:
>🚫 | All of your entries in CHANGELOG.md should be in the **Unreleased** section!

This changelog entry is under the `0.34.3` release heading.  Move it up under the `## [Unreleased]` heading on line 18.  See the very top of the changelog for more direction.

Reply:
@levithomason  can you resolve this conversation now? I fixed it but when I tried resolving it still won't let it merge.